src/Transports.AspNetCore/Assembly.cs

```diff
@@ -0,0 +1,3 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("GraphQL.Server.Transports.AspNetCore.Tests")]
```

Comment:
@Shane32 What I have learnt not so much time ago - https://www.meziantou.net/declaring-internalsvisibleto-in-the-csproj.htm  `InternalsVisibleTo` attribute became "native" since NET5.

Reply:
Are you suggesting we convert to csproj prop?

Reply:
yes

Reply:
Changed.